src/progress/createProgressElement.ts

```diff
@@ -0,0 +1,37 @@
+import { CustomElementDescriptor } from '@dojo/widget-core/customElements';
+import Progress from './Progress';
+
+/**
+ * Configures a Progress web component
+ */
+export default function createProgressElement(): CustomElementDescriptor {
+	return {
+		tagName: 'dojo-progress',
+		widgetConstructor: Progress,
+		attributes: [
+			{
+				attributeName: 'value',
+				value: value => Number(value)
+			},
+			{
+				attributeName: 'max',
+				value: value => Number(value)
+			},
+			{
+				attributeName: 'min',
+				value: value => Number(value)
+			},
+			{
+				attributeName: 'min',
```

Comment:
Duplicate `min` attributes. I think one should be `id`, but just wasn't changed?

Reply:
ah oops, thanks